pkgs/development/python-modules/parsley/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage
+, fetchPypi
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "Parsley";
+  version = "1.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hcd41bl07a8sx7nmx12p16xprnblc4phxkawwmmy78n8y6jfi4l";
+  };
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

Reply:
I hope this suffices... I feel a little bad though.